### ISSUE TITLE
lxc version should show server version too

### ISSUE THREAD

**Issue by markshuttle** [lxc/lxd#4382]
'lxc --version' currently shows only the client version. With LXD remote servers, it would be useful to know the version of those as well. Perhaps something like this:
```
$ lxc version
lxc                3.1.0
localhost          3.0.0
remote.server.com  3.1.1
```

**Comment by stgraber** [comment]
```stgraber@castiana:~/data/code/lxc/lxd (stgraber/master)$ lxc version
Client version: 3.0.0
Server version: 3.0.0.beta7

stgraber@castiana:~/data/code/lxc$ lxc version s-vorash:
Client version: 3.0.0
Server version: 2.21
```

So it will now print your client's version as well as the remote server's version which can be specified as with our other commands.

Listing the version of all remotes is a bit of a problem given that we have users with several hundreds of them, some of which may have high latency and some of which may be offline. It seems more consistent to just list your local and remote version and allow you to specify the remote server if you care about something other than the default remote.

**Comment by stgraber** [comment]
Sent a pull request which addresses this issue. After it's reviewed and merged, the fix will be included in the stable-3.0 branch and be included in the candidate snap as well as in the next batch of cherry-picks for the LXD deb.
